sycl/test/lit.cfg.py

```diff
@@ -90,6 +90,8 @@
 config.substitutions.append( ('%llvm_build_lib_dir',  config.llvm_build_lib_dir ) )
 config.substitutions.append( ('%llvm_build_bin_dir',  config.llvm_build_bin_dir ) )
 
+config.substitutions.append( ('%fsycl-host-only', '-std=c++17 -Xclang -fsycl-is-host -isystem %s -isystem %s -isystem %s' % (config.sycl_include, config.opencl_include_dir, config.sycl_include + '/sycl/') ) )
```

Comment:
I am not sure -std=c++17 needs to be part of fsycl-host-only, but that is probably Ok, if that is intentional and has meaning "using sycl switches assumes using c++17"

Reply:
> if that is intentional and has meaning "using sycl switches assumes using c++17"

That was intentional, we can't compile our headers in c++14 mode, unfortunately. It is probably a bug, but for now I think that it is ok to stick with c++17 explicitly